components/function-controller/README.md

```diff
@@ -27,19 +27,18 @@ To develop the component, use the formulae declared in the [generic](/common/mak
 | **APP_FUNCTION_BUILD_EXECUTOR_IMAGE**               | Full name of the Kaniko executor image used for building Function images and pushing them to the Docker Registry                                                                                                                                                                                             | `gcr.io/kaniko-project/executor:v0.22.0`                                                                                                                 |
-| **APP_FUNCTION_BUILD_CREDS_INIT_IMAGE**             | Full name of the Tekton image used for building Functions. It is run in the Init Container to adjust the Docker credentials to the Kaniko-compatible format.                                                                                                                                                 | `gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/creds-init:v0.12.1`                                                                             |
-| **APP_FUNCTION_DOCKER_ADDRESS**                     | Internal address of the Docker Registry                                                                                                                                                                                                                                                                      | `serverless-docker-registry.kyma-system.svc.cluster.local:5000`                                                                                          |
-| **APP_FUNCTION_DOCKER_EXTERNAL_ADDRESS**            | External address of the Docker Registry                                                                                                                                                                                                                                                                      | `registry.kyma.local`                                                                                                                                    |
+| **APP_FUNCTION_DOCKER_INTERNAL_SERVER_ADDRESS**                     | Internal server address of the Docker Registry                                                                                                                                                                                                                                                                      | `serverless-docker-registry.kyma-system.svc.cluster.local:5000`                                                                                          |
```

Comment:
```suggestion
| **APP_FUNCTION_DOCKER_INTERNAL_SERVER_ADDRESS**                     | Internal server address of the Docker registry                                                                                                                                                                                                                                                                      | `serverless-docker-registry.kyma-system.svc.cluster.local:5000`                                                                                          |
```

Reply:
Either leave `Docker Registry` and I'll change that in the whole file later on or pls change all occurrences of `Docker Registry` into `Docker registry` (lowercase). We talk about a registry in general, so I don't think we should capitalize it. Either way, let me know what you decide to do :)